src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -264,5 +264,39 @@ public static void SafeX509ChainHandle_InvalidHandle_StaticObject()
 
             Assert.Same(firstCall, secondCall);
         }
+
+        [Fact]
+        [OuterLoop( /* May require using the network, to download CRLs and intermediates */)]
```

Comment:
Would this work for Offline as well?  I'm wondering if we should test both, potentially verifying that Offline works successfully after Online has run and hopefully cached whatever it needs to?